.tool-versions

```diff
@@ -0,0 +1 @@
+elixir 1.6.1
```

Comment:
I'm using asdf to manage my elixir version, I added this by mistake - but not sure if we want to specify an explicit version of elixir to use?

Reply:
Let's not add these files, they tend to cause unnecessary churn 👌